src/stubs/LocalizedVagrantfile

```diff
@@ -0,0 +1,23 @@
+require 'json'
+require 'yaml'
+
+VAGRANTFILE_API_VERSION = "2"
+confDir = $confDir ||= File.expand_path("~/.composer/vendor/laravel/homestead")
```

Comment:
I think we're kinda thinking about this in two different ways. I'm saying for the local version you should have "laravel/homestead" as a composer dependency of your main laravel app.